lib/semian/simple_state.rb

```diff
@@ -0,0 +1,49 @@
+module Semian
+  module Simple
+    class State #:nodoc:
+      def initialize
+        @enums = [:closed, :open, :half_open]
+        reset
+      end
+
+      attr_reader :value
+
+      def value=(val)
+        raise ArgumentError unless @enums.include?(val)
+        @value = val
+      end
```

Comment:
Yup. No need to test a method that is not used. Just remove the method and the test althogether.